src/js/angular/templates/modal/add-location.html

```diff
@@ -62,17 +62,17 @@ <h3 class="modal-title">{{'attach.remote.gdb.instance' | translate}}</h3>
         </div>
         <div class="form-group row" ng-if="newLocation.authType === 'signature'">
             <div class="col-md-12">
-                {{'signature.auth.info' | translate}}<a href="{{docBase}}/access-control.html#gdb-authentication"
-                                                        rel="noopener" target="_blank">Access Control / GDB authentication</a>{{'signature.auth.info.last' | translate}}
-            </div>
+            {{'signature.auth.info' | translate}}<a href="{{docBase}}/access-control.html#gdb-authentication"
+                                                    rel="noopener" target="_blank">Access Control / GDB authentication</a>{{'signature.auth.info.last' | translate}}
```

Comment:
This seems to be left untranslated.

Reply:
This is actually a title for sections in the documentation. Not sure if it should be translated